### PR TITLE
New version: Manifold-Release-8 version 8.0.31

### PR DESCRIPTION
**Pull request by rkolka**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25777)

### CODE REVIEW

File: manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+PackageIdentifier: ManifoldSoftware.Manifold-Release-8
+PackageVersion: 8.0.31
+PackageName: Manifold Release 8 Ultimate Edition
```
Comment (OfficialEsco) [inline manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml:3]:
```suggestion
PackageName: Manifold Systems
```

File: manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - IMS
+PackageUrl: https://manifold.net/
+Installers:
+- Architecture: x64
+  InstallerUrl: https://manifoldgis.com/updates/manifold-8.0.31-x64.exe
+  InstallerSha256: c4890604c88e154ea8f47e2c77c85b6658d5b6421938fdc137271e5c6d137fb9
+  InstallerType: exe
+  InstallerSwitches:
+    Silent: /quiet
+    SilentWithProgress: /quiet
```
Comment (OfficialEsco) [inline manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml:23]:
```suggestion
  InstallerType: burn
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml (hunk 1 above)
Comment (rkolka) [reply manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml:3]: "Manifold Release 8" and "Manifold Release 9" This is how Manifold Software calls their products. I'd hesitate to remove "Release" from the name. Though, with latest update, every edition of Manifold 8 is now Ultimate Edition, so perhaps
`PackageName: Manifold Release 8`

 https://manifold.net/info/manifold8.shtml
https://manifold.net/info/manifold9.shtml

Comment (OfficialEsco) [reply manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml:3]: Well... Thats the developers problem, both `Manifold Release 8` and `9` gets installed as `Manifold Systems` in `A`dd or `R`emove `P`rograms which WinGet uses to match the program.

Comment (rkolka) [reply manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml:3]: image
I suggest `PackageName: Manifold System 8` ( and `PackageName: Manifold System 9` for the other)

Comment (OfficialEsco) [reply manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml:3]: The problem with commenting on 2 places about the same thing https://github.com/microsoft/winget-pkgs/pull/25779#discussion_r695549892 😁 Go ahead with the change :)

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/25777#discussion_r695560757

File: manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+License: Copyright (c) Manifold Software Limited
+Moniker: manifold
+ShortDescription: Release 8 is a classic GIS providing seamless desktop, enterprise, programming and web serving capabilities in single integrated product
+Tags: 
+  - gis
+  - map
+  - SQL
+  - ETL
+  - database
+  - IMS
```
Comment (ItzLevvie) [inline manifests/m/ManifoldSoftware/Manifold-Release-8/8.0.31/ManifoldSoftware.Manifold-Release-8.yaml:14]:
```suggestion
- gis
- map
- sql
- etl
- database
- ims
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/25777/files#r698036710

File: manifests/m/ManifoldSoftware/Manifold/8/8.0.31/ManifoldSoftware.Manifold.8.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+PackageIdentifier: ManifoldSoftware.Manifold.8
+PackageVersion: 8.0.31
+PackageName: Manifold 8
+Publisher: Manifold Software Limited
+License: Copyright (c) Manifold Software Limited
+Moniker: manifold
```
Comment (ItzLevvie) [inline manifests/m/ManifoldSoftware/Manifold/8/8.0.31/ManifoldSoftware.Manifold.8.yaml:6]:
```suggestion
Moniker: manifold-8
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/m/ManifoldSoftware/Manifold/8/8.0.31/ManifoldSoftware.Manifold.8.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+PackageIdentifier: ManifoldSoftware.Manifold.8
+PackageVersion: 8.0.31
+PackageName: Manifold 8
```
Comment (ItzLevvie) [inline manifests/m/ManifoldSoftware/Manifold/8/8.0.31/ManifoldSoftware.Manifold.8.yaml:3]:
```suggestion
PackageName: Manifold System 8
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**